src/Compilers/VisualBasic/Portable/Symbols/Tuples/TupleTypeSymbol.vb

```diff
@@ -1125,5 +1125,16 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             Next
         End Sub
 
+        Friend Overrides ReadOnly Property AllRequiredMembers As ImmutableSegmentedDictionary(Of String, Symbol)
+            Get
+                Return _underlyingType.AllRequiredMembers
```

Comment:
>_underlyingType.AllRequiredMembers [](http://example.com/codeflow?start=23&length=34)

I do not think we can simply return members from the underlying type. Are we testing this code path?